### PR TITLE
add environment variables to the task definition

### PR DESCRIPTION
**Pull request by rryke**
needed to expose environment variables in this module. I pulled the variable descriptions from here https://github.com/cloudposse/terraform-aws-ecs-container-definition

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -42,6 +42,7 @@ module "container_definition" {
   container_cpu                = "${var.container_cpu}"
   container_port               = "${var.container_port}"
   healthcheck                  = "${var.healthcheck}"
+  environment                  = "${var.environment}"
```
Comment (osterman) [inline main.tf]: Can you update readme.yaml example with environment variables. Then build readme.

**Comment by rryke** [comment]
doesn't look like that target is in the makefile, am I missing something?

**Comment by osterman** [comment]
(Sorry on phone so using shorthand)

make init 
make readme/deps
make readme

File: README.yaml [hunk 2]
```diff
@@ -80,6 +80,16 @@ usage: |-
     alb_ingress_healthcheck_path = "/healthz"
     alb_ingress_paths            = ["/*"]
     codepipeline_enabled         = "false"
+    environment = [
```
Comment (aknysh) [inline README.yaml]: just a nitpick, but this TF snippet needs `terraform fmt`

File: README.md [hunk 3]
```diff
@@ -43,6 +43,16 @@ module "default-backend-web-app" {
   alb_ingress_healthcheck_path = "/healthz"
   alb_ingress_paths            = ["/*"]
   codepipeline_enabled         = "false"
+  environment = [
```
Comment (aknysh) [inline README.md]: just a nitpick, but this TF snippet needs `terraform fmt`
well be regenerated after updating in `README.yaml` and then rebuilding README

```
make init
make readme/deps
make readme
```

File: variables.tf [hunk 4]
```diff
@@ -85,6 +85,11 @@ variable "launch_type" {
   default     = "FARGATE"
 }
 
+variable "environment" {
+  description = "The environment variables for the task definition. This is a list of maps"
```
Comment (aknysh) [inline variables.tf]: can you add
type        = "list"

(and do `terraform fmt`)

**Comment by rryke** [comment]
the environment variable was pulled exactly as they we're being used in the downstream module. I'm cool with "list" just wanted to keep consistency in the module usage sense. 👍

**Pull request merged**